doc/docusaurus/src/components/HomepageFeatures.tsx

```diff
@@ -11,7 +11,7 @@ type FeatureItem = {
 const FeatureList: FeatureItem[] = [
   {
     title: 'Write clean, maintainable code',
-    image: '/img/gears-gear-svgrepo-com.svg',
+    image: '/solid_lints/img/gears-gear-svgrepo-com.svg',
```

Comment:
so any of the [documentation's suggested](https://docusaurus.io/docs/static-assets#referencing-your-static-asset) options haven't worked for me. i've decided to just hardcode url since it's not supposed to be changed frequently